handlers/content.ts

```diff
@@ -39,9 +49,15 @@ export const getUserFeed: Handler<{}> = async (c: Context<{}, {}, T.Paths.GetUse
   return res.status(200).json(response);
 };
 
-export const getFeed: Handler<{}> = async (c: Context<{}, {}, T.Paths.GetFeed.QueryParameters>, _req, res) => {
+export const getFeed: Handler<object> = async (
+  c: Context<object, object, T.Paths.GetFeed.QueryParameters>,
+  _req,
+  res,
+) => {
   // Return only items from who the user follows
-  const msaId = c.security.tokenAuth.msaId || (await getMsaByPublicKey(c.security.tokenAuth.publicKey));
+  const msaId =
+    c.security.tokenAuth.msaId ||
+    (await getMsaByPublicKey(c.security.tokenAuth.publicKey));
```

Comment:
Using this in a couple of places. do we want to make it a helper?

Reply:
Not for now, but something for the next version for sure to do better.